en/components/grids_templates/column_resizing.md

```diff
@@ -167,7 +166,126 @@ column.autosize();
 #### Pinned columns resizing
 
 Pinned columns can also be resized. However, resizing is limited so that the overall width of the pinned columns container cannot become larger than 80% of the overall @@igComponent width.
-Again, if you try to autosize a pinned column and the new width will cause the pinned columns container to exceed those 80% of the overall @@igComponent width, autosizing will be discarded. We just want to make sure that the unpinned columns are always visible and available to the user!
+Again, if you try to autosize a pinned column and the new width will cause the pinned columns container to exceed those 80% of the overall @@igComponent width, auto sizing will be discarded. We just want to make sure that the unpinned columns are always visible and available to the user!
+
+### Styling
+To get started with the styling of the @@igComponent column resize line, we need to import the index file, where all the theme functions and component mixins live:
+
+```scss
+// grid-resize-line-styling-sample.scss
+@import '~igniteui-angular/lib/core/styles/themes/index';
+```
+
+The simplest approach to achieve this is to create a new theme that extends the [`igx-grid-theme`]({environment:sassApiUrl}/index.html#function-igx-grid-theme) and accepts many parameters as well as the `$resize-line-color` parameter.
+
+``` scss
+$blue-color: #0288D1;
+$custom-grid-theme: igx-grid-theme(
+    $resize-line-color: $blue-color
+);
```

Comment:
The first demonstration of the theme creation should be the the most hard-coded, without CSS variables.
`$custom-grid-theme: igx-grid-theme(
    $resize-line-color:  #0288D1
);`
This should be enough.